Dockerfile

```diff
@@ -14,7 +14,7 @@ ADD docker/entrypoint.sh /bin/entrypoint.sh
 RUN chmod 770 /bin/entrypoint.sh
 ENTRYPOINT ["/bin/entrypoint.sh"]
 
-HEALTHCHECK CMD lua -l socket -e 'assert(socket.connect(os.getenv"SNIKKET_TWEAK_INTERNAL_HTTP_INTERFACE" or "127.0.0.1",os.getenv"SNIKKET_TWEAK_INTERNAL_HTTP_PORT" or "5280"))'
+HEALTHCHECK CMD /usr/bin/prosodyctl shell "portcheck ${SNIKKET_TWEAK_INTERNAL_HTTP_INTERFACE:-127.0.0.1}:${SNIKKET_TWEAK_INTERNAL_HTTP_PORT:-5280}"
```

Comment:
Would that work in a shell? I would've expected `/usr/bin/prosodyctl shell portcheck "${SNIKKET_TWEAK_INTERNAL_HTTP_INTERFACE:-127.0.0.1}:${SNIKKET_TWEAK_INTERNAL_HTTP_PORT:-5280}"` (note the unescaped shell between `portcheck` and the other argument.

Reply:
`prosodyctl shell "single command"` runs exactly that single command in the shell, while if you give it more than one argument it goes and composes a `foo:bar("baz", "moo")` style command out of it.

Reply:
Ahh I forgot about the magic which adds a colon. Could you add explicit parentheses to make it more obvious?

Reply:
It's a simple command, like `help`, there are no parentheses, just a single line.